config/cmake/config/VXLIntrospectionConfig.cmake

```diff
@@ -656,21 +655,6 @@ endif()
 
 # Check C++ <cmath> first, where the C++11 standard says these must be.
 include(${CMAKE_CURRENT_LIST_DIR}/CheckCXXExpressionCompiles.cmake) ## From VTK
-check_cxx_expression_compiles("std::isnan(0.0)" "cmath" VXL_HAS_STD_ISNAN)
```

Comment:
These removals should speed up initial configure time. Yeah!

Reply:
I hope so.  There are many more config time checks that are now meaningless.  Will remove as time permits.